src/WebJobs.Script/Binding/ServiceBus/ServiceBusScriptBindingProvider.cs

```diff
@@ -66,7 +66,7 @@ public static void ValidateAccessRights(AccessRightsValidator validator, string
                 }
                 catch (UnauthorizedAccessException)
                 {
-                    throw new UnauthorizedAccessException($"Service Bus Trigger binding '{bindingName}' requires a connection string with Manage AccessRights for correct triggering and scaling behavior when running in a consumption plan.");
+                    traceWriter.Warning($"Service Bus Trigger binding '{bindingName}' requires a connection string with Manage AccessRights for correct triggering and scaling behavior when running in a consumption plan.");
```

Comment:
This log only goes to the host log, right? And we don't surface that anywhere in the portal? If so users simply aren't going to see this ever - I think we were discussing that before.